tests/faker.py

```diff
@@ -0,0 +1,24 @@
+"""
+This file holds reusable fake stuff for mocking the tests.
+"""
+
+FAKE_CHANNEL = 'C12942JF92'
```

Comment:
I actually meant holding all the credentials in a dict:

```
fake_creds = {
    'FAKE_CHANNEL': 'C12942JF92',
}
```

All is good though.